src/ansys/tools/visualization_interface/backends/pyvista/pyvista.py

```diff
@@ -131,6 +129,8 @@ def __init__(
         self._origin_colors = {}
 
         # Enable the use of trame if requested and available
+        from ansys.tools.visualization_interface.backends.pyvista.trame_local import _HAS_TRAME
```

Comment:
How are you getting this flag created?

Because if you are using something a `try` and `except` like [it seems you do](https://github.com/ansys/ansys-tools-visualization-interface/blob/e9eea916d6f12105ef2b4278c14ba23b8041e0ff/src/ansys/tools/visualization_interface/backends/pyvista/trame_local.py#L23-L30) :

https://github.com/ansys/ansys-tools-visualization-interface/blob/e9eea916d6f12105ef2b4278c14ba23b8041e0ff/src/ansys/tools/visualization_interface/backends/pyvista/trame_local.py#L23-L30

Maybe you are importing trame (maybe not all and that could explain the time reduction). Can you try something like this?

https://stackoverflow.com/questions/14050281/how-to-check-if-a-python-module-exists-without-importing-it

Reply:
I actually need to make a conditional import in any case for the trame modules, they might not be installed and the module can be imported. However, I can use the approach you propose to avoid importing the whole trame module where it is not needed in the `pyvista` internal module. Thanks for the suggestion 🙂

Reply:
I see you did the changes, removing the import. Thank you ! Did you test the speed again?

(I should apply myself my own advice and do this in PyMAPDL 😅)

Reply:
It's more or less the same, but it makes sense, since I was lazy importing the required module. In any case, the code it's better this way.

![image](https://github.com/user-attachments/assets/bb0b26e9-a0ea-4bf9-a469-c8025f26f440)

Reply:
I guess if Trame is not importing much stuff on `__init__` it wont make a difference. But thanks a lot for trying!